src/functions/heroFunctions/getStoredPlayers.tsx

```diff
@@ -0,0 +1,14 @@
+import { PLAYERS_KEY } from "../../typesAndConstants/constants";
+
+export default function getStoredPlayers(): { [key: string]: HeroToJSON } {
```

Comment:
if you have characterManager this can be in it as well. and every action with localStorage will be managed by it